build.gradle

```diff
@@ -225,14 +225,25 @@ subprojects { subproject ->
     }
 
     dependencies {
-        compile 'org.slf4j:slf4j-api:1.6.4' // libthrift 0.5 otherwise pins 1.5.x
-
         testCompile 'junit:junit:4.12'
         testCompile 'org.mockito:mockito-all:1.10.19'
         testCompile "org.scalatest:scalatest_${scalaInterfaceVersion}:2.2.5"
     }
 }
 
+// Ensures libthrift 0.5.0 isn't accidentally in the classpath
+subprojects { subproject ->
+    configurations.all {
+        resolutionStrategy {
+            eachDependency { DependencyResolveDetails details ->
+                if (details.requested.group == 'org.apache.thrift') {
+                    details.useVersion '0.9.0' // Scrooge calls TBinaryProtocol.setReadLength
```

Comment:
Actually we can't because the method scrooge uses is removed in 0.9.1
(unless it was too late last night)